package.json

```diff
@@ -85,7 +85,10 @@
     "cssnano": "^5.0.7",
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
+    "express-jwt": "^6.1.0",
```

Comment:
@IgorChaikin I think this package is unused in project, you can use only `jsonwebtoken`